products/BellHybrid/apps/application-bell-relaxation/windows/RelaxationRunningLoopWindow.cpp

```diff
@@ -72,16 +74,15 @@ namespace
         return battery;
     }
 
-    std::string adjustDisplayedTitle(const std::string &title)
+    std::string adjustDisplayedTitle(const UTF8 &title, const std::uint32_t maxCharsInLine)
     {
-        if (title.length() <= maxPossibleCharsToDisplay) {
+        const auto maxTitleLength =
+            static_cast<std::uint32_t>(std::round(static_cast<float>(maxCharsInLine) * charsMultiplier));
```

Comment:
Try to use `std::floor` instead of `std::round`. I'm not sure but maybe the `charsMultiplier` won't be necessary.